lib/graphql/card.ts

```diff
@@ -0,0 +1,101 @@
+import { Card as CardModel, Query } from "./schema";
+import { Model } from "./model";
+import { ResultPage } from "./resultPage";
+import { CardOptions } from "./types";
+
+const CARD_FIELDS = `
+  id
+  status
+  type
+  canceledAt
+  expirationDate
+  orderedAt
+  holder
+  formattedExpirationDate
+  maskedPan
+  settings {
+    contactlessEnabled
+    cardNotPresentLimits {
```

Comment:
I'm not sure about exposing cardLimits directly here, this would trigger 2 calls from Kontist backend to Solaris (one for cardNotPresentLimits and one for cardPresentLimits). Limits will be displayed on a separate screen and doesn't have to be fetched by default

Reply:
so, do you think I should add them for example to `get({})` method only? (the one who gets one card by ID), and generic `fetch()` method would not return them, does it sound correct?

Reply:
or maybe we will have another method specifically for these limits later? like `client.models.card.getLimits({ cardId })`?

I think I will remove these limits from this PR for now so that it becomes mergeable, and we can adjust code later on to expose limits somewhere

Reply:
So, for now I removed the limits entirely, we will see how we decide to introduce them alter on.

Here it is: 08bd85269213474117035208c67d728dbb60d883

Reply:
your solution looks good to me, we can extend SDK with new method when we will be close to work with card limit screens :)